spec/models/air_quality_observation_spec.rb

```diff
@@ -1,11 +1,14 @@
 require 'rails_helper'
+require_relative '../helpers/aqi_api_stubs_helper'
 
 describe AirQualityObservation do
+  include AqiApiStubsHelper
+
   before do
     create_list(:air_quality_observation, 10)
   end
 
-  describe 'closest' do
+  describe '#closest' do
```

Comment:
Nice. I like this convention just because it's easier to see what stuff does from the specs.